test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -357,6 +357,12 @@
     ("aten::_nested_view_from_buffer_copy.out", datetime.date(2023, 5, 1)),
     ("aten::_nested_view_from_buffer_copy", datetime.date(2023, 5, 1)),
     ("aten::_nested_view_from_buffer", datetime.date(2023, 5, 1)),
+    # These ops were moved to python under the c10d_functional namespace
+    ("aten::wait_tensor", datetime.date(9999, 1, 30)),
```

Comment:
is there precedent for using 9999 instead of a real date? i get that there is probably no valid fc/bc concern since these ops probably aren't used anywhere in prod yet